ols/customize/ols/prompts.py

```diff
@@ -96,3 +96,56 @@
 {{query}}
 Response:
 """
+
+# {{query}} is escaped because it will be replaced as a parameter at time of use
+TOPIC_SUMMARY_PROMPT_TEMPLATE = """
+Instructions:
+- You are a topic summarizer
+- Your job is to extract precise topic summary from user input
+
+For Input Analysis:
+- Scan entire user message
+- Identify core subject matter
+- Distill essence into concise descriptor
+- Prioritize key concepts
+- Eliminate extraneous details
+
+For Output Constraints:
+- Maximum 5 words
+- Capitalize only significant words (e.g., nouns, verbs, adjectives, adverbs).
+- Do not use all uppercase - capitalize only the first letter of significant words
+- Exclude articles, prepositions, and punctuation (e.g., "a," "the," "of," "on," "in")
```

Comment:
it  may create confusion when excluding "of" "on" "in"

Reply:
is this for generating keywords for RAG retrieval?
what is the upstream PR for this?